wiki/Help_centre/Upgrading_to_lazer/en.md

```diff
@@ -153,35 +153,71 @@ That said, individual scores and beatmaps can be exported from lazer and manuall
 
 #### If I set a score on lazer, will it show on my profile?
 
-Scores will show under "recently played" but not in "best performance" yet.
+Yes, but it won't show in "best performance" with "lazer mode" turned off on the website.
 
-#### If I set a score on lazer, will it give performance points?
+It additionally not show in "first place ranks" regardless for now.
 
-The "recently played" section of profile pages shows scores set in lazer with performance points calculated, and the total amount can be seen when hovering over the regular performance point total.
+#### If I set a score on lazer, will it give performance points?
 
-Additionally, [lazer.ppy.sh](https://lazer.ppy.sh/home), an experimental instance of the osu! website, shows all totals and rankings with lazer scores accounted for.
+Yes, with the caveat that scores may be deleted in the future.
```

Comment:
Honestly at this point maybe we can take this out, but let's wait a few more weeks.